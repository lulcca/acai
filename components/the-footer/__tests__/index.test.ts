import { beforeEach, describe, expect, it } from 'vitest';
import TheFooter from '@/components/the-footer/index.vue';
import { mount } from '@vue/test-utils';

describe('Components > TheFooter', () => {
  let wrapper: ReturnType<typeof mount>;

  beforeEach(() => wrapper = mount(TheFooter));

  it('should match snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });

  it('should render the copyright text correctly', () => {
    expect(wrapper.find('span').text()).toBe('Â© 2025 lulcca');
  });
});
