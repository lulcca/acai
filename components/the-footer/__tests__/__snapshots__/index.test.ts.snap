// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Components > TheFooter > should match snapshot 1`] = `"<footer class="grid opacity-75 place-items-start"><span class="font-major"> Â© 2025 lulcca </span></footer>"`;
